src/hotspot/share/opto/superword.cpp

```diff
@@ -3074,7 +3069,7 @@ void VTransform::adjust_pre_loop_limit_to_align_main_loop_vectors() {
   const bool is_sub = iv_scale * iv_stride > 0;
 
   // 1.1: con
-  Node* xbic = igvn().intcon(is_sub ? -con : con);
+  Node* xbic = phase()->intcon(is_sub ? -con : con);
```

Comment:
What changed that you need/want `PhaseIdealLoop*` here instead of `PhaseIterGVN*`?